diff --git a/src/llamafactory/data/mm_plugin.py b/src/llamafactory/data/mm_plugin.py
--- a/src/llamafactory/data/mm_plugin.py
+++ b/src/llamafactory/data/mm_plugin.py
@@ -42,9 +42,11 @@ from ..extras.packages import is_pillow_available, is_pyav_available, is_transfo
 
 
 if is_pillow_available():
-    from PIL import Image
+    from PIL import Image, ImageFile
     from PIL.Image import Image as ImageObject
 
+    ImageFile.LOAD_TRUNCATED_IMAGES = True
+
 
 if is_pyav_available():
     import av
